modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,129 @@
+
+= stable-{product-version}
+
+The stable channel, containing only GA releases, will contain updates on a time delay
+as they are gradually rolled out to customers based on data from our SRE teams,
+support services, and pre-production and production environments that participate in
+our connected customer program, rather than being immediately available as they are
+in the fast channel. For patch and CVE fixes this can range from several hours to a
+day and allows an extra period of assessment in how the software performs. This
+channel also includes the upgrade from the previous 4.3 release.
+
+= Fast and Stable Channel Usage and Strategies
+
+The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+GA updates as soon as they are available or allowing Red Hat to control the rollout
+of those updates. If issues are detected during rollout, upgrades to that version may
+be blocked in both the fast and stable channels, and a new version may be introduced
+that will be the new preferred upgrade target.
+
+Customers can improve this process by configuring pre-production systems on the
+fast channel, production systems on the stable channel, and participating in Red
+Hat’s connected customer program. This program allows Red Hat to observe the impact
+of updates on your specific hardware and software configurations. Future releases may
+improve or alter the pace at which updates move from the fast to the stable channels.
+
+If issues are discovered with an upgrade between patch levels, Red Hat may withdraw
+that suggested upgrade for affected versions. A newer patch would become available
+in the appropriate channels and be suggested for upgrade.
+
+= Upgrade Version Paths
+
+{product-title} maintains an upgrade recommendation service that understands the
+version of {product-title} you have installed as well as the path to take within
+the channel you choose to get you to the next release. You can imagine seeing the
+following in the fast-{product-version} channel:
+
+* {product-version}.0
+* {product-version}.1
+* {product-version}.3
+* {product-version}.4
+
+The service only recommends upgrades that have been tested and have no known issues.
+If you are on {product-version}.1 and {product-title} is allowing you to select
+{product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise,
+the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no
+longer suggests upgrading to a known vulnerable version. If an issue is found that
+results in a new version being retracted from the recommendations, Red Hat will
+release a new version that is capable of upgrading from all necessary versions,
+including the retracted version.
+
+= Disconnected Clusters
+
+Customers which have chosen to not be connected to Red Hat and are curating their
+own {product-title} container image content manually should consult the Red Hat
```

Comment:
I am not sure of `curating` is the correct word here. Somehow it gives an impression that customers can create  own container image. But that's not supported right?

Reply:
> Somehow it gives an impression that customers can create own container image. But that's not supported right?

Yeah, although I think we want "container image content manually" -> "update recommendations", because what they're owning is the update graph, not the release images.

I don't have strong feelings on "curating" vs. support, although it does imply that the official Red Hat graph is something that customers might want to polish.  While that is occasionally true, any polishing should happen [in the graph-data repo][1], where everyone can submit PRs, customers included.  What restricted-network users are doing is owning maintenance on their update graph, which gives all the power, responsibility, and headache that forking a project usually gives.  "curating" -> "maintaining" or some such might do a better job of getting that balance across.  But I'm personally fine leaving "curating" if folks like it more.

[1]: https://github.com/openshift/cincinnati-graph-data/